IntelliJ Project/src/main/java/com/globant/musicstore/entity/Song.java

```diff
@@ -34,7 +34,7 @@
     private Album album;
 
     @OneToOne(cascade = {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH})
-    @JoinColumn(name = "cat_genre_id", referencedColumnName = "id")
-    private Cat_Genre catGenre;
+    @JoinColumn(name = "cat_genre_id", referencedColumnName = "cat_genre_id")
```

Comment:
I prefer to let referencedColumName = "id" because it is the way is saved in the DB.

Reply:
![image](https://user-images.githubusercontent.com/20212610/134455373-490f8113-d857-4bda-ac3d-e5f21e069259.png)
It's actually cat_genre_id according to the script 😅

Reply:
Okay, I said that because as I mencioned in the meeting, I don´t use a data base script. But anyway, I'll check that in my local. Ty